librz/bin/format/mach0/mach0.c

```diff
@@ -2751,12 +2752,12 @@ static int walk_exports(struct MACH0_(obj_t) * bin, RExportsIterator iterator, v
 			RZ_FREE(next);
 			goto beach;
 		}
-		if (SZT_ADD_OVFCHK((size_t)tr, (size_t)trie) || (size_t)tr + trie >= end) {
+		if (tr >= (ut64)(end - trie)) {
```

Comment:
why not the OVFCHK check?

Reply:
Simplified it even more. Explicit overflow-checking is just not necessary anymore now.